src/main/java/seedu/address/storage/JsonAdaptedTask.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.animal.Task;
+import seedu.address.model.animal.TaskList;
+
+
+
+/**
+ * A Jackson-friendly adapted version of the {@link TaskList} model class.
+ */
+public class JsonAdaptedTask {
+    private final String description;
+    private final String taskStatus;
+
+    /**
+     * Constructs a {@code JsonAdaptedTask} with the given Task details.
+     * This constructor is used internally by Jackson to deserialize {@link Task} JSON objects.
+     */
+    @JsonCreator
+    public JsonAdaptedTask(String description, String taskStatus) {
```

Comment:
Consider renaming `taskStatus` to `isDone` for easier mapping of attributes.

Reply:
Will do so as I work on the storage!